easybuild/easyconfigs/p/Python/Python-2.7.12-iccifort-2016.3.210-GCC-5.4.0-2.26-bare.eb

```diff
@@ -0,0 +1,35 @@
+name = 'Python'
+version = '2.7.12'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'iccifort', 'version': '2016.3.210-GCC-5.4.0-2.26'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
```

Comment:
`optarch` and `opt` are enabled by default, no need to specify these (we need to do a cleanup sweep for those sometime...)